### PR TITLE
Bugfix and some QoL

### PR DESCRIPTION
**Pull request by klemenvon**
Some changes to how we handle terminal colors internally. Fixed bug where student numbers would get cut off too far when prettifying user IDs. These changes are only tested for staff account with no quota overruns. Should be fully tested before deploying new version. Do that before accepting the PR.

### CODE REVIEW

File: pgquota/pgquota [hunk 1]
```diff
@@ -1,13 +1,43 @@
 #!/usr/bin/env python3
```
Comment (bedroge) [inline pgquota/pgquota]:
```suggestion
#!/usr/bin/python3
```
Let's stick to the OS Python here, as discussed on Slack.

File: pgquota/pgquota [hunk 2]
```diff
@@ -1,242 +1,328 @@
...
+    # Lets try this implementation
+    # Create an object that contains all quota information for one directory for one user
+    # Then we can print it or not
+    def __init__(self, group, directory, debug_flag):
+        self.group = group
+        self.directory = directory
+        self.raw = subprocess.check_output(['lfs', 'quota', '-g', str(self.group), str(self.directory), '-h']).decode(
+            "utf-8")
+        if debug_flag:
+            print("{}Debug Raw{}".format(colors.get('red'), colors.get('end')))
```
Comment (bedroge) [inline pgquota/pgquota]: `colors` has to be replaced here.

File: pgquota/pgquota [hunk 3]
```diff
@@ -1,242 +1,328 @@
...
+            # print(Col.red("Debug Raw"))
+            print(self.raw)
+            print("{}End of Debug Raw{}".format(colors.get('red'), colors.get('end')))
```
Comment (bedroge) [inline pgquota/pgquota]: and here

File: pgquota/pgquota [hunk 4]
```diff
@@ -1,242 +1,328 @@
...
+            self.fusage = int(self.ref[5][:-1])
+        else:
+            self.fusage = int(self.ref[5])
+        self.falloc = int(self.ref[6])
+        self.flimit = int(self.ref[7])
+        self.fgrace = self.ref[8]
+        if self.fgrace == "none":
+            self.fgrace = self.fgrace.upper()
+        if debug_flag:
+            print("{}Debug object variables{}".format(colors.get('red'), colors.get('end')))
```
Comment (bedroge) [inline pgquota/pgquota]: and here

File: pgquota/pgquota [hunk 5]
```diff
@@ -1,242 +1,328 @@
...
+            print(
+                "Disk Usage: {}	{}\nDisk Allocated: {}	{}\nDisk Limit: {}	{}\nFile Usage: {}	{}\nFile Allocated: {}	{}\nFile Limit: {}	{}\nDisk Grace: {}	{}\nFile Grace: {}	{}".format(
+                    self.dusage, type(self.dusage), self.dalloc, type(self.dalloc), self.dlimit, type(self.dlimit),
+                    self.fusage, type(self.fusage), self.falloc, type(self.falloc), self.flimit, type(self.flimit),
+                    self.dgrace, type(self.dgrace), self.fgrace, type(self.fgrace)))
+            print("{}End Debug ojbect variables{}".format(colors.get('red'), colors.get('end')))
```
Comment (bedroge) [inline pgquota/pgquota]: and here too.

File: pgquota/pgquota [hunk 6]
```diff
@@ -1,242 +1,328 @@
...
+
+# colors = {
+# 	'blue' : '\033[94m', # some terminals don't support this so it just looks 'normal'
+# 	'red'  : '\033[91m',
+# 	'bold' : '\033[1m',
+# 	'warn' : '\033[93m', # not sure yet what colour this is
+# 	'end'  : '\033[0m' # if this isn't at the end of the string, the formatting can overflow onto more lines of text
+# }
+
+class Col():
```
Comment (bedroge) [inline pgquota/pgquota]: I'd just name it Color?

**Review by bedroge — CHANGES_REQUESTED** [review]
I've made some style changes, and requested some more changes.
We should also remove all the commented lines (old functionality, e.g. the `colors` dict).

**Comment by klemenvon** [comment]
Addressed all the issues and entirely removed old (commented out) code or reduce fluff. Updated the RPM spec file as well, so all this needs is one more good look and then we can install it.

**Review by klemenvon — COMMENTED** [review]
Made requested changes and deleted old, commented out code.

**Review by bedroge — APPROVED** [review]

**Pull request merged**